codejail_service/urls.py

```diff
@@ -19,30 +19,12 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
 """
 
-import os
-
-from auth_backends.urls import oauth2_urlpatterns
-from django.conf import settings
-from django.contrib import admin
 from django.urls import include, re_path
-from rest_framework_swagger.views import get_swagger_view
 
 from codejail_service.apps.api import urls as api_urls
 from codejail_service.apps.core import views as core_views
 
-admin.autodiscover()
-
-urlpatterns = oauth2_urlpatterns + [
-    re_path(r'^admin/', admin.site.urls),
+urlpatterns = [
     re_path(r'^api/', include(api_urls)),
-    re_path(r'^api-docs/', get_swagger_view(title='codejail-service API')),
```

Comment:
Could this be useful in a development environment?

Reply:
Hmm... I suppose. I could bring some of these things back in for consistency with other IDAs.

Reply:
Looks like this maybe requires a DB.